query/src/test/java/org/infinispan/query/blackbox/ClusteredCacheTest.java

```diff
@@ -544,57 +542,57 @@ public void testClear() throws Exception {
       StaticTestingErrorHandler.assertAllGood(cache1, cache2);
    }
 
-   public void testFullTextFilterOnOff() throws Exception {
-      prepareTestData();
-
-      CacheQuery<Person> query = createQuery(cache1, "blurb:'eats'");
-      FullTextFilter filter = query.enableFullTextFilter("personFilter");
-      filter.setParameter("blurbText", "cheese");
-
-      assertEquals(1, query.getResultSize());
-      List<Person> result = query.list();
-
-      Person person = result.get(0);
-      assertEquals("MiniGoat", person.getName());
-      assertEquals("Eats cheese", person.getBlurb());
-
-      //Disabling the fullTextFilter.
-      query.disableFullTextFilter("personFilter");
-      assertEquals(2, query.getResultSize());
-      StaticTestingErrorHandler.assertAllGood(cache1, cache2);
-   }
-
-   public void testCombinationOfFilters() throws Exception {
-      prepareTestData();
-
-      person4 = new Person();
-      person4.setName("ExtraGoat");
-      person4.setBlurb("Eats grass and is retired");
-      person4.setAge(70);
-      cache1.put("ExtraGoat", person4);
-
-      CacheQuery<Person> query = createQuery(cache1, "blurb:'eats'");
-      FullTextFilter filter = query.enableFullTextFilter("personFilter");
-      filter.setParameter("blurbText", "grass");
-
-      assertEquals(2, query.getResultSize());
-
-      FullTextFilter ageFilter = query.enableFullTextFilter("personAgeFilter");
-      ageFilter.setParameter("age", 70);
-
-      assertEquals(1, query.getResultSize());
-      List<Person> result = query.list();
-
-      Person person = result.get(0);
-      assertEquals("ExtraGoat", person.getName());
-      assertEquals(70, person.getAge());
-
-      //Disabling the fullTextFilter.
-      query.disableFullTextFilter("personFilter");
-      query.disableFullTextFilter("personAgeFilter");
-      assertEquals(3, query.getResultSize());
-      StaticTestingErrorHandler.assertAllGood(cache1, cache2);
-   }
+//   public void testFullTextFilterOnOff() throws Exception {
```

Comment:
Are these tests obliterated for now with the hope to resurrect them later ?

Reply:
This is something that leaks from Hibernate Search API: the capacity to annotate a filter in the entity and enable or disable it at query time. In the Ickle world, the equivalent would be to not use a predicate when not required. WDYT?

Reply:
Indeed. let's leave them commented out for now. I have no idea what to do with them.